docs/features/logforwarding/multiline-error-detection.adoc

```diff
@@ -0,0 +1,92 @@
+
+== DetectMultilineErrors
+Enables multi-line error detection of container logs
+
```

Comment:
Can we add a big, fat bolded not there are performance implications to enabling this feature.  There may be results associated with the initial issue where we can make a statement?

Reply:
If that helps us with support going forward, yes.  How about:

_"Please be aware that enabling this feature could have performance implications and may require additional computing resources or alternate logging solutions.  We recommend monitoring your system's performance after enabling the feature to ensure that it continues to operate at optimal levels"_

Reply:
I pushed an update with only the first sentence.

Reply:
wfm